src/Microsoft.Sbom.Api/Workflows/SbomToolWorkflow.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.Sbom.Api.Workflows
     /// <summary>
     /// Defines the workflow steps for the drop validation action.
     /// </summary>
-    public class DropValidatorWorkflow : IWorkflow
+    public class SbomToolWorkflow : IWorkflow
```

Comment:
I think this workflow should better be called `SbomValidationWorkflow`. It is not the root workflow as it may appear, it is a workflow for validation process along with `SBOMGenerationWorkflow` that lives in the same folder as describes generation process.

Reply:
Also if we can prefer "SbomXxx" to "SBOMXxx" that would be more consistent with coding standards.

Reply:
@ksigmund I'd prefer SbomXxx but to be consistent with the actual code I'll use SBOMXxx. We can later do a renaming pass to convert SBOM to Sbom in the code

Reply:
Yea, I would also prefer to rename to `SbomXxx` to standardize the code